src/paper/serializers/paper_serializers.py

```diff
@@ -888,7 +888,7 @@ def get_first_preview(self, paper):
         return None
 
     def get_score(self, paper):
-        return paper.unified_document.score
+        return paper.calculate_score()
```

Comment:
There seems to be a disconnect between unified document score and a document's real score right now. All of our other document types (posts, questions, hypotheses) currently use `obj.calculate_score` right now.

In the future, we should definitely be using `unified_document.score`, but right now it seems like it's not up to date since our vote module refactor.